lib/pry/config/default.rb

```diff
@@ -115,7 +115,7 @@ class Default
           Pry::Config.from_hash(Pry::Command::Ls::DEFAULT_OPTIONS)
         },
         completer: proc {
-          require "pry/input_completer"
+          require_relative "../input_completer"
```

Comment:
we could do this, or just require 'pry/input_completer.rb' in pry.rb / pry_class.rb. i don't mind which we choose.

Reply:
I would require from `pry.rb` since it's the first place to check for other requires.